source/read/distinct.txt

```diff
@@ -0,0 +1,152 @@
+.. _c-distinct:
+
+==============================
+Retrieve Distinct Field Values
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, unique, code example
+
+Overview
+--------
+
```

Comment:
S: I think it's a little more clear / more consistent with other guides if you introduce the page topic first, then give an example. The [C++ driver](https://www.mongodb.com/docs/languages/cpp/cpp-driver/upcoming/read/distinct/#overview) does this, for example.